src/javascript/app/common/content_visibility.js

```diff
@@ -74,12 +74,17 @@ const ContentVisibility = (() => {
         const visible_classname = 'data-show-visible';
         const mt_company_rule   = 'mtcompany';
 
+        const map_account_type = {
+            'maltainvest': 'financial',
+        };
+
         document.querySelectorAll('[data-show]').forEach(el => {
             const attr_str              = el.dataset.show;
             const { is_exclude, names } = parseAttributeString(attr_str);
             const rule_set              = new Set(names);
 
-            const rule_set_has_current = rule_set.has(current_landing_company_shortcode);
+            const rule_set_has_current = rule_set.has(current_landing_company_shortcode) ||
+                Client.hasAccountType(map_account_type[names]);
```

Comment:
Will this cause unwanted display issues in other places?
i.e. to display `maltainvest` contents for `malta` and `iom` clients. (Since they have another account by maltainvest)

Reply:
yes it will 😅 I added this to handle virtual landing companies. i guess by for virtual accounts, we'll just show the default llogo altho clients might have MF accounts?

Reply:
The current logged-in account is the one that matters.